web-console/4.0-designs/user/user.md

```diff
@@ -0,0 +1,19 @@
+# User Menu
+
+## Copy Login Command
+![pic 0](img/copy-login-command-1.png)
+
+- Copy Login Command is the first option under the user menu in the masthead.
+
+![pic 1](img/copy-login-command.png)
+
+- Clicking this option opens a modal.
+- The oc login command is presented in conjunction with the user's unique token.
+- A warning is presented above the command to remind users not to share their token.
+
+![pic 3](img/copy-login-command-3.png)
+
+- Users can copy the command and token by clicking the copy icon to the right of the field.
+- Hovering over the button will show a tooltip "copy to clipboard" which will then change to say "copied" after the user clicks.
+- This is the same behavior used on the Secrets page inside the console.
```

Comment:
not sure if we want to, but should we link off to that documentation?  Maybe it's overkill